starfyre/compiler.py

```diff
@@ -2,15 +2,30 @@
 from pathlib import Path
 
 
-def get_fyre_files(project_dir):
+def get_fyre_files(project_dir):   
     fyre_files = []
     for file in os.listdir(project_dir):
         if file.endswith(".fyre"):
             fyre_files.append(file)
     return fyre_files
 
+def resolve_css_import(import_statement, working_directory):
+    """Read a css file and save it's content to a list"""
+    css_content = []    
+    if "" in import_statement:
+        css_file_name = import_statement.replace('"', "'")        
+    css_file_name = css_file_name.split("'")[1]   
+    if css_file_name.startswith("."):
+        css_file_name = css_file_name.replace(".", str(working_directory), 1)
```

Comment:
If you use Path from pathlib(https://docs.python.org/3/library/pathlib.html) , the resolution will be implemeted by internal functions. It is always a better practice to use internal functions wherever possible.